NuGet/Get-BcNuGetPackageId.ps1

```diff
@@ -0,0 +1,54 @@
+<# 
+ .Synopsis
+  PROOF OF CONCEPT PREVIEW: Get Business Central NuGet Package Id from Publisher, Name and Id
+ .Description
+  Get Business Central NuGet Package Id from Publisher, Name and Id
+ .OUTPUTS
+  string
+  Package Id
+ .PARAMETER packageIdTemplate
+  Template for package id with placeholders for publisher, name, id and version
+ .PARAMETER publisher
+  App Publisher (will be normalized)
+ .PARAMETER name
+  App name (will be normalized)
+ .PARAMETER id
+  App Id (must be a GUID)
+ .PARAMETER version
+  App Version
+ .EXAMPLE
+  Get-BcNuGetPackageId -publisher 'Freddy Kristiansen' -name 'Bing Maps PTE' -id '165d73c1-39a4-4fb6-85a5-925edc1684fb'
+#>
+function Get-BcNuGetPackageId {
+    Param(
+        [Parameter(Mandatory=$false)]
+        [string] $packageIdTemplate = '{publisher}.{name}.{id}',
+        [Parameter(Mandatory=$true)]
+        [string] $publisher,
+        [Parameter(Mandatory=$true)]
+        [string] $name,
+        [Parameter(Mandatory=$true)]
+        [string] $id,
+        [Parameter(Mandatory=$false)]
+        [string] $version = ''
+    )
+
+    try { $id = ([GUID]::Parse($id)).Guid } catch { throw "Package id must be a valid GUID: $id" }
+    $nname = [nuGetFeed]::Normalize($name)
+    $npublisher = [nuGetFeed]::Normalize($publisher)
+    if ($nname -eq '') { throw "Package name is invalid: '$name'" }
+    if ($npublisher -eq '') { throw "Package publisher is invalid: '$publisher'" }
+    
+    $packageId = $packageIdTemplate.replace('{id}',$id).replace('{name}',$nname).replace('{publisher}',$npublisher).replace('{version}',$version)
+    if ($packageId.Length -ge 100) {

```

Comment:
Add a comment on why the shortening is needed. Maybe also include it in the doc above.